src/objects/control.ts

```diff
@@ -16,6 +25,57 @@ export function objectifyDescriptions(descs: ExecJSON.ControlDescription[] | { [
   return descs || {}
 }
 
+/**
+ * Represents a Control object with various properties and methods to manipulate and convert it.
+ * 
+ * @class Control
+ * @property {string} id - The unique identifier for the control.
+ * @property {string | null} [title] - The title of the control.
+ * @property {string | null} [code] - The code associated with the control.
+ * @property {string | null} [describe] - Additional description content for the control.
+ * @property {string | null} [desc] - The default description of the control.
+ * @property {Object.<string, string | undefined>} descs - Additional descriptions for the control.
+ * @property {number} [impact] - The impact value of the control.
+ * @property {string} [ref] - A reference string for the control.
+ * @property {(string | { ref?: string; url?: string; uri?: string; })[]} [refs] - An array of references for the control.
+ * @property {Object.<string, string | string[] | Record<string, string[]>[] | boolean | undefined | null>} tags - Tags associated with the control.
+ * 
+ * @constructor
+ * @param {Partial<Control>} [data] - An optional partial object of type Control to initialize the instance with.
+ * 
+ * @method toUnformattedObject
+ * Converts the current Control object into an unformatted object.
+ * The method flattens the object, processes its string properties, and then unflattens it back into a Control object.
+ * @returns {Control} A new Control object created from the unformatted data.
+ * 
+ * @method toString
+ * Converts the control object to a string representation in a specific format.
+ * The resulting string includes:
+ * - The control ID.
+ * - The title, if present.
+ * - The default description, if present.
+ * - Additional descriptions, if present.
+ * - The impact value, if present.
+ * - References, if present.
+ * - Tags, if present.
+ * - Additional describe content, if present.
+ * @returns {string} The string representation of the control object.
+ * 
+ * @method toRuby
```

Comment:
I think you added the toString and toRuby documentation below where the method is defined as well. Do you want to keep it in both places or just put it with the method definition?